yal/main.py

```diff
@@ -128,8 +128,19 @@ async def process_message(self, message):
             message.session_event = Message.SESSION_EVENT.RESUME
             self.state_name = feedback_state
 
+        # Replies to template push messages
+        payload = utils.get_by_path(
+            message.transport_metadata, "message", "button", "payload"
+        )
+        if payload and payload.startswith("state_") and payload in dir(self):
```

Comment:
I think we need to remove the `and payload in dir(self)` from here because some of the states won't exist in the codebase yet.

Reply:
That's probably what we want though isn't it? We don't want to change what we're doing if it will result in us sending the user an error, we rather stick with existing states for testing (which is why I added the test state, so that we have something to test with). I also wanted to try and make this test as close to the actual code as possible